Utilities/bootstrap

```diff
@@ -1125,6 +1127,9 @@ def main():
             symlink_force(os.path.join(args.foundation_path, 'libFoundation.so'),
                 libswiftdir)
             if args.libdispatch_build_dir:
+                symlink_force(os.path.join(args.libdispatch_build_dir,
+                                           'libBlocksRuntime.so'),
+                              libswiftdir)
```

Comment:
Why do we need this symlink?

Reply:
Without that symlink, the tests tended to fail.  libdispatch would not be found, but by keeping the symlink there, which is one of the paths that linker/loader consult, the tests are happier.